app/src/main/java/com/kms/cura/constant/EventConstant.java

```diff
@@ -14,6 +14,7 @@
     public static final String REGISTER_SUCCESS = "REGISTER_SUCCESS";
     public static final String REGISTER_FAILED = "REGISTER_FAILED";
     public static final String SEARCH_SUCCESS = "SEARCH_SUCCESS";
+    public static final String UPDATE_SUCCESS = "UPDATE_SUCCESS";
```

Comment:
Should be more specific with events. In this case it should be UPDATE_PATIENT_HEALTH_SUCCESS